lib/src/controls/surfaces/expander.dart

```diff
@@ -262,21 +266,25 @@ class ExpanderState extends State<Expander>
           );
         },
       ),
-      SizeTransition(
-        sizeFactor: _controller,
-        child: Container(
-          width: double.infinity,
-          padding: const EdgeInsets.all(16.0),
-          decoration: BoxDecoration(
-            border: Border.all(
-              color: theme.resources.cardStrokeColorDefault,
+      // CONTENT
+      Flexible(
```

Comment:
Why?

Reply:
This enables us to put two expanded expanders in one constrained space, otherwise there is an issue. I'm trying to modify the code further to enable the same two expanders to then resize dynamically as they currently aren't doing so. I'll update here when I've worked that out. Thanks.

Reply:
Sure!

Reply:
The work I was intending to do was too long and can be implemented in the future. As it stands, this change enables us to put ListViews directly in Expander contents. Its most useful where Expanders are meant to occupy the full screen space. I would say this commit is final then and you can see if it meets the requirements to be merged. Thanks!